dmgroups.tex

```diff
@@ -135,29 +135,47 @@ \subsection{DM Leadership Team \label{sect:dmlt}}
 
 \subsection{DM Change Control Board \label{sect:dmccb}}
 
-The DMCCB has responsibility for issues similar to those of the LSST Change Control Board, but with its scope restricted to the DM Subsystem.
+The DMCCB has responsibility for issues similar to those of the LSST Change Control Board, but focused on the  DM Subsystem.
```

Comment:
What's the intending meaning of this change? Do we expect the DMCCB to be sometimes looking at changes outside DM?

Reply:
Maybe just add: "... but focused only on the DM ..."

Reply:
I still don't understand the reason for this change in wording. The old version was quite clear; the new one seems to be adding ambiguity for no gain.